hicommonwealth/commonwealth
# Added contest card to the topic view

## Link to Issue
Closes: #8110 

## Description of Changes
- modified existing contest card and reused it in the topic view

## Test Plan
- create contest within `topic-expample`
- go to `/discussions/topic-example`
- the banner should be visible until the contest ends

![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/ed95b392-9c23-43ff-9c95-e7e019b06e45)

## Deployment Plan

1. n/a
